dashboard/config/locales/en.yml

```diff
@@ -370,8 +370,8 @@ en:
     print_certificate: "Print Certificate"
     announcement_title: "Announcements"
     announcement_view_all: "View all"
-    announcement_message: "You can now add App Lab projects to a website or blog using the new embed code feature. Just click on the \"Share\" button, go to \"Show advanced options,\" and copy the embed code.  You can then paste the embed code into any web page, blog, or site that uses HTML to share your App Lab projects!"
-    announcement_button: "Try it now"
+    announcement_message: "Many of you gave feedback that when students work in pairs or small teams, only one student gets credit for their progress. We've created the ability to let students connect their accounts while they work together, so you, the teacher, can track completed progress for both individuals and teams."
+    announcement_learn_more_button: "Learn more"
```

Comment:
Instead of having to translating this button string back and forth, I'm going to keep this learn more button and add new strings when we have new call to actions like "Try it now"